spec/lib/action_kit/helper_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+describe ActionKit::Helper do
+  before do
+    ENV['AK_PASSWORD'] = 'ak_password'
+    ENV['AK_USERNAME'] = 'ak_username'
```

Comment:
The issue that we were having is that ENV Variables were being changed during tests then not reset, meaning that other tests which tested parts of the code that were dependent on those same ENV variables were entering unexpected code paths and failing.

Setting the values to `nil` in the after is probably OK, but it's still a state change that could have unexpected results down the line. It feels a bit like tests which have unexpected database states - not necessarily bad but things could get sloppy.